client/src/component/Pages.jsx

```diff
@@ -0,0 +1,263 @@
+import React from 'react';
+import styles from '../componentsStyles/PagesStyle.jsx';
```

Comment:
```suggestion
import {PageButton, ArrowButtons, NumberInButton, SelectedButton, SelectDiv, PageButtonList, PageButtonUL} from '../componentsStyles/PagesStyle.jsx';
```
You can destructure the styles here to make the render method a little clearer.